### ISSUE TITLE
dimension not consistent

### ISSUE THREAD

**Issue by Wlllllllly** [nicetomeetu21/CA-LDM#2]
Hello,
Thank you for your awsome work, I have some question:
In the step 4:Train LDM2D_refiner. Are you using the 3D as the condition of the 2D diffusion? The code:
out = self.model(x=torch.cat([x,c], dim=1), timesteps=t)
x is the 3D and c is the 2D, how can it concate?

Looking forward for your reply!

**Comment by nicetomeetu21** [comment]
Hi,
You can check the paper and the datamodule file. Here x and c are both 2D frames of a 3D latent representation cube.

**Comment by Wlllllllly** [comment]
> Hi, You can check the paper and the datamodule file. Here x and c are both 2D frames of a 3D latent representation cube.

OK! I will check the paper! Thanks!
Additionally, have you released the uniaxial SR module code?

**Comment by nicetomeetu21** [comment]
Yes, it is a simple inplementation based on the decoder in VQGAN. Here it is.
https://github.com/nicetomeetu21/CA-LDM/blob/16f01afeea1e2045e907f7d1ef3ac8bcabd9c526/train_NHVQVAE.py#L119

**Comment by Wlllllllly** [comment]
> Yes, it is a simple inplementation based on the decoder in VQGAN. Here it is.
> 
> https://github.com/nicetomeetu21/CA-LDM/blob/16f01afeea1e2045e907f7d1ef3ac8bcabd9c526/train_NHVQVAE.py#L119
OK! Thank you !
